deployment/src/main/java/io/quarkiverse/quinoa/deployment/packagemanager/Commands.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkiverse.quinoa.deployment.packagemanager;
+
+import java.util.Collections;
+
+import io.quarkus.runtime.LaunchMode;
+
+interface Commands {
```

Comment:
Maybe `PackageManagerCommands` or rename `Command` to `ExecCommand` (to make it a bit more readable). WDYT?